packages/plugins/rubygem-chunky_png/chunky_png-1.3.10.gem

```diff
@@ -1 +1 @@
-../.git/annex/objects/W5/j8/SHA256E-s352768--9c08320a5af529b922c2a2102cd2e2f15325375cc352baeaa45e2e9e22ede215.10.gem/SHA256E-s352768--9c08320a5af529b922c2a2102cd2e2f15325375cc352baeaa45e2e9e22ede215.10.gem
\ No newline at end of file
+../../../.git/annex/objects/W5/j8/SHA256E-s352768--9c08320a5af529b922c2a2102cd2e2f15325375cc352baeaa45e2e9e22ede215.10.gem/SHA256E-s352768--9c08320a5af529b922c2a2102cd2e2f15325375cc352baeaa45e2e9e22ede215.10.gem
```

Comment:
What is this?

Reply:
git annex 'magic' that snuck in. no idea.  removing